Makefile

```diff
@@ -37,3 +39,9 @@ dist: dev-env
 .PHONY: set-version
 set-version:
 	python set-version.py
+
+.PHONY: html-docs
+html-docs: ## Build html docs
```

Comment:
I know I'm nitpicking but what's with the double #?

Reply:
Also I think just docs would be a better name than html-docs.

Reply:
the double # are from mlrun :)))